app/view_models/v1/disbursement.rb

```diff
@@ -0,0 +1,38 @@
+module V1
+  class Disbursement < BaseViewModel
+    include ActionView::Helpers::NumberHelper
```

Comment:
Do we need `NumberHelper` if we are using `NumberTo.pounds` from lib ?

Reply:
:(